packages/codegen/query/control.ts

```diff
@@ -270,11 +270,58 @@ export const GET_CONTROL_SELECT_OPTIONS = gql`
           refCode
           category
           subcategory
-          standard {
-            shortName
-          }
+          referenceFramework
+        }
+      }
+    }
+  }
+`
+
+export const GET_CONTROL_CATEGORIES = gql`
+  query GetControlCategories {
+    controlCategories
+  }
+`
+
+export const GET_CONTROL_SUBCATEGORIES = gql`
+  query GetControlSubcategories {
+    controlSubcategories
+  }
+`
+
+export const GET_CONTROLS_PAGINATED = gql`
+  query GetControlsPaginated($where: ControlWhereInput, $after: Cursor) {
+    controls(where: $where, after: $after) {
+      totalCount
+      edges {
+        node {
+          id
+          refCode
+          category
+          subcategory
+          referenceFramework
         }
       }
+      pageInfo {
+        hasNextPage
+        endCursor
+      }
+    }
+  }
+`
+
+export const GET_CONTROL_BY_ID_MINIFIED = gql`
+  query GetControlByIdMinified($controlId: ID!) {
+    control(id: $controlId) {
+      id
+      refCode
+      standard {
+        id
+        shortName
+      }
```

Comment:
```suggestion
      referenceFramework
      standardID
```

shouldn't be a big performance hit on a single query - but referenceFramework is the standard short name so you can always pull this without a subquery with the above

Reply:
fixed